pkg/migration/controllers/migration.go

```diff
@@ -184,6 +184,27 @@ func (m *MigrationController) Handle(ctx context.Context, event sdk.Event) error
 		var terminationChannels []chan bool
 		var err error
 
+		// Fail the migration if the current domain is inactive
+		clusterDomains, err := m.Driver.GetClusterDomains()
```

Comment:
Can we retry this 4-5 times with a delay of 2-3 seconds? So that even if we hit the window where PX is restarting you will eventually get the cluster domains?